src/validator-integration.js

```diff
@@ -34,14 +35,24 @@ export function maybeValidate(win) {
   }
 
   /** @const {!Element} */
-  const s = win.document.createElement('script');
+  const validatorScript = win.document.createElement('script');
   // TODO(@cramforce): Introduce a switch to locally built version for local
   // development.
-  s.src = `${urls.cdn}/v0/validator.js`;
-  s.onload = () => {
-    win.document.head.removeChild(s);
-    /* global amp: false */
-    amp.validator.validateUrlAndLog(filename, win.document, getMode().filter);
-  };
-  win.document.head.appendChild(s);
+  validatorScript.src = `${urls.cdn}/v0/validator.js`;
+
+  const examinerScript = win.document.createElement('script');
+  examinerScript.src = `${urls.cdn}/examiner0.js`;
+
+  Promise.all([
```

Comment:
Worth delaying the validation before examiner is loaded? Maybe better split into separate calls? I.d. probably factor out a shared function, e.g.:
```
function loadScript(src) {
  const el = ...
  const promise = loadPromise(el);
  promise.catch().then(() => head.removeChild(el));
  head.appendChild(el);
  return promise;
}

loadScript(validator).then(() => {validateUrlAndLog()})
loadScript(examiner).then(() => ...)
```

?